src/com/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -950,28 +1049,30 @@
         (select-keys migrations pending))))
 
 (defn migrate!
-  "Migrates database to the latest schema version. Does nothing if database is
```

Comment:
I just noticed while reviewing another PR that we already have a fast-forward-to-migration! in master.  Given that, we may want to back out this change to migrate!.